src/Engine/Systems/PhysicsSystem.cpp

```
@@ -185,15 +195,42 @@ void PhysicsSystem::syncECSData() {
+		data.position = transform.position;
+		data.orientation = transform.rotation;
+		data.scale = transform.scale;
+
+		data.velocity = rigidBody.velocity;
+		data.acceleration = rigidBody.acceleration;
+		data.mass = rigidBody.mass;
+
+		if (m_orbitTrajectories.count(entityID) && m_orbitTrajectories.at(entityID).dirty) {
+			data.orbitVertices = m_orbitTrajectories.at(entityID).vertices;
+			m_orbitTrajectories.at(entityID).dirty = false;
+		}
```

Cppcheck (2.20.0)

[style] The function 'origin' is never used.

(unusedFunction)